### DISCUSSION TITLE
UNOFFICIAL apt repository for installing on Linux/debian* distros

### DISCUSSION THREAD

**Discussion by sbrl** [mbrlabs/Lorien#313] [Show and tell]
Heya!

I really like this tool from the little I've used it so far, and given it's infinite nature it seems like a cross between draw.io and Inkscape, which is very fun indeed! I do like the default bright neon colours! My biggest hope is that #63 will be implemented 'cause my handwriting *sucks*.

In the meantime for all you Linux users out there on Debian-based distributions like Ubuntu (AUR my beloved!), I just thought I'd say that I have setup automated packaging of Lorien in my personal apt repository (which also repackages Blender, and the Krita + DrawPile AppImages among others).

You can find it here:

<https://apt.starbeamrainbowlabs.com/>

Basically, follow the instructions there to setup the apt repository, and then install the package like so:

```bash
sudo apt update
sudo apt install lorien-sbrl
```

...name has `-sbrl` just in case official packaging is done later down the line to avoid conflicts.

Basically it just downloads the GitHub release and repackages it with a `.desktop` file, the logo, etc with [fpm](https://fpm.readthedocs.io/)

I have set my [CI system](https://github.com/ohwgiles/laminar/) up to automatically check every week (i.e. via `/etc/cron.weekly`) for updates. If updates are found, they will be added to the repository 2am the following day. CI logs are private due to the way my infrastructure is setup.

If you notice updates aren't going through (allow ~1 week + 1 day), then just poke me on any platform you like and I'll fix the script.

Hope this is helpful for someone!

**Comment by GutPuncher** [comment]
Would love to Lorien in the official Ubuntu packages one day!

**Reply by sbrl** [reply]
That would be the dream! Basically just requires a debian package maintainer to take notice. I'm theory your could become a debian package maintainer if got had the time, but I don't so I just do this instead.

I don't think my repackaging would live up to the debian standards lol

**Comment by mbrlabs** [comment]
Very cool, thank you!

**Reply by sbrl** [reply]
Tysm! 🌟
